# HG changeset patch
# Parent 4ef0d33459e27e7ce669256cfdfda9d7f6b5925a
ARMAsmBackend.cpp  was moved to lib/Target/ARM/MCTargetDesc/

diff -r 4ef0d33459e2 -r 9649c959e4ce lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp
--- a/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp	Mon Aug 08 15:04:54 2011 -0700
+++ b/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp	Mon Aug 08 15:13:46 2011 -0700
@@ -387,6 +387,12 @@
   void ApplyFixup(const MCFixup &Fixup, char *Data, unsigned DataSize,
                   uint64_t Value) const;
 
+  // @LOCALMOD-BEGIN
+  unsigned getBundleSize() const {
+    return (OSType == Triple::NativeClient) ? 16 : 0;
+  }
+  // @LOCALMOD-END
+
   MCObjectWriter *createObjectWriter(raw_ostream &OS) const {
     return createELFObjectWriter(new ARMELFObjectWriter(OSType), OS,
                               /*IsLittleEndian*/ true);
